js/tidex2.js

```diff
@@ -0,0 +1,960 @@
+            'last': last,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': this.safeString (ticker, 'avg'),
+            'baseVolume': this.safeString (ticker, 'vol_cur'),
+            'quoteVolume': this.safeString (ticker, 'vol'),
+            'info': ticker,
+        }, market, false);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetTickers (params);
+        const result = this.safeValue (response, 'result');
+        //
+        //     {
+        //         "code":200,
+        //         "success":true,
+        //         "message":"",
+        //         "result":{
+        //             "YFI_USDT":{
+        //                 "at":1646289676,
+        //                 "ticker":{
+        //                     "name":"yfi_usdt",
+        //                     "bid":"21937.47155034",
+        //                     "ask":"21966.13380147",
+        //                     "open":"21449.08",
+        //                     "high":"22619.38",
+        //                     "low":"21193.59",
+        //                     "last":"21968.03",
+        //                     "vol":"188.4452385",
+        //                     "deal":"4075398.065557751",
+        //                     "change":"2"
+        //                 }
+        //             }
+        //         }
+        //     }
+        //
+        return this.parseTickers (result, symbols);
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        const response = await this.publicGetTicker (this.extend (request, params));
+        const result = this.safeValue (response, 'result');
+        //
+        //     {
+        //         "code":200,
+        //         "success":true,
+        //         "message":"",
+        //         "result":{
+        //             "name":"YFI_USDT",
+        //             "bid":"21607.842223",
+        //             "ask":"21635.50514299",
+        //             "open":"21530.31",
+        //             "high":"22619.38",
+        //             "low":"21193.59",
+        //             "last":"21644.23",
+        //             "volume":"188.4555124",
+        //             "deal":"4075612.142786252",
+        //             "change":"1"
+        //         }
+        //     }
+        //
+        return this.parseTicker (result, market);
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit; // default = 1, max = 100
```

Comment:
the default is actually 50, and the max is actually 1000, the min 1 so line 435 should be

```JavaScript
request['limit'] = limit; // default = 50, max = 1000, min = 1
```